app/frontend/src/theme.ts

```diff
@@ -86,7 +87,7 @@ const themeOptions: ThemeOptions = {
   shape: {
     borderRadius,
     navPaddingDesktop: spacing(10),
-    navPaddingMobile: spacing(navBarHeight),
+    navPaddingMobile: spacing(navBarHeight + navBarPadding),
```

Comment:
Had to add a little of bit padding back, as the header with the back button in the single discussion was getting clipped under the nav bar. This was also a problem for the group chat view, so I reckon that's why the padding was there before. The padding is a bit less than before though and it seems fine with the map page.

Reply:
I think it's better for pages to handle this individually, for the same reason we are now usually avoiding putting margins on components.. Most pages do so with `<PageTitle>` which has padding. The map page now has a gap which breaks the fullscreen effect.